src/platform/platform_linux.cpp

```diff
@@ -148,10 +165,18 @@ bool mp::platform::is_alias_supported(const std::string& alias, const std::strin
 
 bool mp::platform::is_remote_supported(const std::string& remote)
 {
+    const auto& driver = utils::get_driver_str();
+    // TODO: Remove this constraint once we support other image remotes for hte LXD backend
+    if (driver == QStringLiteral("lxd") && (remote != "release" && remote != "daily"))
```

Comment:
Shouldn't this also test for `remote.isEmpty()`?

Reply:
Yep, will fix.

Reply:
Yeah, it shouldn't be ;)